### PR TITLE
[Workflow] Add info about MermaidDumper

### PR DESCRIPTION
**Pull request by eFrane**
This updates the documentation for symfony/symfony#40171.

### CODE REVIEW

File: workflow/dumping-workflows.rst [hunk 1]
```diff
@@ -9,6 +9,7 @@ them as SVG or PNG images. First, install any of these free and open source
 applications needed to generate the images:
 
 * `Graphviz`_, provides the ``dot`` command;
+* `Mermaid CLI`, provides the ``mmdc``command;
```
Comment (wouterj) [inline workflow/dumping-workflows.rst]:
```suggestion
* `Mermaid CLI`_, provides the ``mmdc``command;
```

(the `` `...`_ `` syntax is used for links in reStructuredText)

This also requires adding this line to the bottom of the article:

```
.. _`Mermaid CLI`: https://github.com/mermaid-js/mermaid-cli
```

File: workflow/dumping-workflows.rst [hunk 2]
```diff
@@ -76,6 +84,9 @@ Colors can be defined as:
 * a color name from `PlantUML's color list`_;
 * an hexadecimal color (both ``#AABBCC`` and ``#ABC`` formats are supported).
 
+*Caveat*: The Mermaid dumper does not support coloring the arrow heads
+with ``arrow_color`` as there is no support in Mermaid for doing so.
```
Comment (wouterj) [inline workflow/dumping-workflows.rst]:
```suggestion
.. note::

    The Mermaid dumper does not support coloring the arrow heads
    with ``arrow_color`` as there is no support in Mermaid for doing so.
```

File: workflow/dumping-workflows.rst [hunk 3]
```diff
@@ -9,6 +9,7 @@ them as SVG or PNG images. First, install any of these free and open source
 applications needed to generate the images:
 
 * `Graphviz`_, provides the ``dot`` command;
+* `Mermaid CLI`, provides the ``mmdc``command;
 * `PlantUML`_, provides the ``plantuml.jar`` file (which requires Java).
```
Comment (wouterj) [inline workflow/dumping-workflows.rst:14]: We also tell people about new features using a `versionadded` block:
```suggestion

.. versionadded:: 5.3

    The ``mermaid`` dump format was introduced in Symfony 5.3.
```

**Review by wouterj — APPROVED** [review]
Hi @eFrane. You are today's hero for taking the time to document a future you proposed on the code!

I've left 3 markup related comments (reStructuredText has some overly complex syntax). Don't worry about these, we can also fix them while merging.

File: workflow/dumping-workflows.rst (hunk 3 above)
Comment (eFrane) [reply workflow/dumping-workflows.rst:14]: While I fixed the other issues, I left out this one as the feature PR is not yet merged and although there's still plenty of time until 5.3 I erred on the cautious side.

**Comment by wouterj** [comment]
Thank you @eFrane and congratulations with your first contributions! :)

Fyi, I've added the versionadded box while merging the PR.

**Pull request merged**